releasenotes.txt

```diff
@@ -0,0 +1,315 @@
+BISDN Linux
+4.8.0 Release Notes
+Date: 2022-09-05
+
+! New in this release
+- added Edgecore EPS201 (AS4630-54TE) as supported platform
+- updated to Linux kernel 5.15.64
+- bundle MSTPD state as well in collect-debug-info
+- allow ignoring failing commands in bundle-debug-info
+- allow arbitrary argument order for bundle-debug-info
```

Comment:
I don't think this is important enough to mention here.

Reply:
removed